### DISCUSSION TITLE
Markdown - How to Have Link Open in New Window?

### DISCUSSION THREAD

**Discussion by NotGovernor** [requarks/wiki#3020] [Help / Questions] [answered]
I searches issues and saw that this was added last year but I do not see how in the documentation. Is there a markdown tag for this or should I just use an HTML link?

Thanks.

**Comment by NGPixel** [comment, marked as answer]
You need to use HTML code to set the target property:

```
<a href="/link-url" target="_blank">Link Text</a>
```
